pebblo/app/utils/utils.py

```diff
@@ -282,3 +283,24 @@ def delete_directory(app_path, app_name=None):
         logger.exception(message)
     finally:
         return result
+
+
+def get_current_time():
+    """
+    Return current time in isoformat
+    :return:
+    """
+    return datetime.now().isoformat()
+
+
+def timeit(func):
```

Comment:
Can timeit function have a condition in the beginning because everytime we will calculate the time for each function but not log it.. So can it only run when the deubg mode is on else it just returns from the first line itself